server/Episememe.Application.Tests/FeatureTests/GetFavoriteMedia/WhenGettingFavoriteMedia.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Episememe.Application.DataTransfer;
+using Episememe.Application.Features.GetFavoriteMedia;
+using Episememe.Application.Interfaces;
+using Episememe.Application.Tests.Helpers;
+using Episememe.Domain.Entities;
+using FluentAssertions;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Threading;
+using Xunit;
+
+namespace Episememe.Application.Tests.FeatureTests.GetFavoriteMedia
+{
+    public class WhenGettingFavoriteMedia : IDisposable
```

Comment:
You could add many more cases to test:
- there are favorite entries in a database for more than 1 user. The handler should only return entries for one particular user, and ignore others
- what if there's no favorite entries at all. Does it return an empty list or throw?